compiler/z/codegen/S390Instruction.hpp

```diff
@@ -2550,7 +2550,10 @@ class S390RSInstruction : public TR::S390RegInstruction
 
    int16_t                 _sourceImmediate;
    int8_t                  _maskImmediate;
-   bool                    _hasMask;
+   /// _hasMaskImmediate evaluates to true if a value for the _maskImmediate field has been specified, false otherwise
+   bool                    _hasMaskImmediate;
```

Comment:
I don't think this is a very descriptive comment. The name of the field already implies relation to `_maskImmediate`. The comment doesn't give any indication as to _why_ we have the two `_sourceImmediate` and `_maskImmediate` fields. Perhaps a better comment may describe why we make the distinction:

```
/// Determines whether a mask value was specified during the construction of this object, indicating an RS instruction of RS-b format where the third operand is a mask
bool _hasMaskImmediate;

/// Determines whether an immediate value was specified during the construction of this object, indicating an RS instruction of RS-a format where the third operand is a memory reference interpreted as an immediate value.
bool _hasSourceImmediate;

Reply:
Updated here: 387883a